driver/others/init.c

```diff
@@ -415,7 +402,7 @@ static void numa_mapping(void) {
     }
   }
   for (i = 0; i < MAX_BITMASK_LEN; i++)
-    cpu_count += popcount(common -> node_info[current_node][i] & common -> avail[i]);
+    cpu_count += popcount(common -> node_info[current_node][i]);
```

Comment:
The revised cpu_count now represents the number of logical cores per NUMA node, whereas before the revision it reflected the number of physical cores. This change may cause errors in condition checks like if (numprocs <= cpu_count), where tasks that originally required execution across two nodes might now be incorrectly judged as runnable within a single node